app/src/main/java/comcmput301f16t01/github/carrier/Requests/DriverViewRequestActivity.java

```diff
@@ -239,6 +236,7 @@ public void getRoadAsync() {
      * This AsyncTask updates the road on the map and maps a route between two points. This is
      * so that it does not lock up the UI thread or try to make a network connection on it.
      */
+    // see code attribution
```

Comment:
I was just saying that because at the top of every file I write in the Javadocs "See code attribution in Wiki" with a link to it. I can make the change if you want but I will do it tomorrow morning. :)